src/Features/ERDDAP/waterLevel/chart/largeTimeSeriesChart.tsx

```diff
@@ -123,7 +123,11 @@ export function LargeTimeSeriesWaterLevelChart({
         </XAxis>
         <YAxis
           softMin={softMin}
-          softMax={floodThresholds ? floodThresholds?.Major?.minValue + 3 : softMax[unitSystem]}
+          softMax={
+            floodThresholds
+              ? floodThresholds?.Major?.minValue + dataConverter.convertToNumber(0.5, unitSystem)
```

Comment:
Do we want to get the max of all the thresholds, in case there isn't Major, or one of the impacts is higher? (often with USGS, I've seen Major being some point that is hit every 10 years or so, and there are much higher impacts).

Reply:
Good thinking. I'll refactor to get the highest.